sale_cutoff_time_delivery/models/stock_picking.py

```diff
@@ -17,6 +19,37 @@ class StockPicking(models.Model):
         search="_search_cutoff_time_diff",
         store=False,
     )
+    expected_delivery_date = fields.Datetime(compute="_compute_expected_delivery_date")
+
+    def _compute_expected_delivery_date(self):
+        """Computes the expected delivery date.
+        In some cases, the order expected_date and commitment_date
```

Comment:
```suggestion
        """Computes the expected delivery date.

        In some cases, the order expected_date and commitment_date
```
(empty line between the summary and detailed description)